modules/plotting/src/properties/tickproperty.cpp

```diff
@@ -167,17 +170,17 @@ TickProperty::TickProperty(std::string_view identifier, std::string_view display
     : CompositeProperty(identifier, displayName, invalidationLevel, semantics)
     , majorTicks_("majorTicks", "Major Ticks")
     , minorTicks_("minorTicks", "Minor Ticks") {
-    addProperty(majorTicks_);
-    addProperty(minorTicks_);
+
+    addProperties(majorTicks_, minorTicks_);
```

Comment:
Could be just get rid of this property? Would be nice to not have to unfold that level since it is only 2 nodes in it?

Reply:
Removed this property since it was used nowhere.